templates/environment/partials/environment-manager-role.yml

```diff
@@ -193,7 +193,8 @@ EnvironmentManagerRole:
           Effect: Allow
           Action: [
             "ec2:DescribeSubnets",
-            "ec2:DescribeSecurityGroups"
+            "ec2:DescribeSecurityGroups",
+            "ec2:DescribeNetworkInterfaces"
```

Comment:
ooh this is gonna be trickkyyy, customers with existing environments won't get this new permission so trying to show them the IP addresses will return errors :(

Let's add this `Describe` permission but I think we should behave in such a way that on any errors trying to get the IP addresses we don't fail the CLI. Instead, we should just ignore the error and not display the IP addresses in that situation.

Reply:
Can you try deploying an environment without this change, and then calling `task run` with your changes to validate that we don't error out and the CLI just doesn't display the IP addresses?

Reply:
Updated! Also posted screenshots in PR comment^